cuqi/distribution/_posterior.py

```diff
@@ -78,9 +78,15 @@ def geometry(self, value):
         else:
             self._geometry = self.prior.geometry
             
-    def logpdf(self, x):
+    def logpdf(self, *args, **kwargs):
         """ Returns the logpdf of the posterior distribution"""
-        return self.likelihood.logd(x)+ self.prior.logd(x)
+        return self.likelihood.logd(*args, **kwargs)+ self.prior.logd(*args, **kwargs)
+
+    def get_conditioning_variables(self):
+        return self.prior.get_conditioning_variables()
+
+    def get_parameter_names(self):
```

Comment:
In these two methods, `get_conditioning_variables` and `get_parameter_names`

Is it possible that the likelihood has additional conditioning variables ?

Reply:
This is a good point. I will add a check during the initializer that ensures the likelihood parameters match a distribution.